ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml

```diff
@@ -192,6 +192,14 @@ tests:
       DESTINATION_BRANCH: community-4.14
       SOURCE_BRANCH: master
     workflow: windows-ci-fastforward
+- as: nutanix-e2e-operator
+  always_run: false
+  optional: true
+  steps:
+    cluster_profile: nutanix
+    test:
+    - ref: windows-e2e-operator-test
+    workflow: openshift-e2e-nutanix-ovn
```

Comment:
This workflow does not have a ovn-hybird-manifest step. Do we need to add that?

Reply:
Yes we will need that for Windows support. Ref: https://docs.openshift.com/container-platform/4.12/networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.html